### PR TITLE
chore: Make EE opt-out

### PR DESCRIPTION
**Pull request by JanCizmar**

### CODE REVIEW

File: webapp/src/views/projects/translations/BatchOperations/types.ts [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
+//   | 'mark_as_translated'
+//   | 'task_create'
+//   | 'task_add_keys'
+//   | 'task_remove_keys'
+//   | 'add_tags'
+//   | 'remove_tags'
+//   | 'change_namespace'
+//   | 'copy_translations'
+//   | 'clear_translations'
+//   | 'export_translations';
```
Comment (stepan662) [inline webapp/src/views/projects/translations/BatchOperations/types.ts:17]: Why?

File: webapp/src/views/projects/translations/BatchOperations/BatchSelect.tsx [hunk 2]
```diff
@@ -21,114 +17,17 @@ const StyledSeparator = styled('div')`
 `;
 
 type Props = {
-  value: BatchActions | undefined;
-  onChange: (value: BatchActions | undefined) => void;
+  value: string | undefined;
+  onChange: (value: string | undefined) => void;
```
Comment (stepan662) [inline webapp/src/views/projects/translations/BatchOperations/BatchSelect.tsx:21]: This should stay typed

File: webapp/src/plugin/getEe.ts
Comment (stepan662) [inline webapp/src/plugin/getEe.ts:1]: Is this file necessary? If `eePlugin.local` doesn't exist, doesn't it fail completely anyway?

File: webapp/src/fixtures/pluginAdder.ts
Comment (stepan662) [inline webapp/src/fixtures/pluginAdder.ts:1]: Not sure I follow this. Why is the file named pluginAdder and the name of the function is createAdder?

File: webapp/scripts/prepareEe.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
+// if src/ee exists copy the webapp/src/eePlugin.ee.tsx to webapp/src/eePlugin.local.tsx
+// else copy the webapp/src/eePlugin.oss.tsx to webapp/src/eePlugin.local.tsx
```
Comment (stepan662) [inline webapp/scripts/prepareEe.js:2]: It's actually a symlink, does it need to be a symlink?

File: webapp/src/component/App.tsx [hunk 4]
```diff
@@ -96,7 +58,12 @@ export class App extends React.Component {
     globalContext.actions?.setGlobalError(error as GlobalError);
     throw error;
   }
+
   render() {
+    const {
+      billing: { GlobalLimitPopover },
+    } = getEe();
```
Comment (stepan662) [inline webapp/src/component/App.tsx:65]: Can't we just do `import {GlobalLimitPopover} from tg.ee` or something like that?

**Review by Anty0 — CHANGES_REQUESTED** [review]

**Review by stepan662 — APPROVED** [review]

**Review by Anty0 — APPROVED** [review]

**Pull request merged**